bin/node/runtime/src/lib.rs

```diff
@@ -186,6 +189,47 @@ impl pallet_utility::Trait for Runtime {
 	type IsCallable = ();
 }
 
+parameter_types! {
+	// One storage item; key size 32, value size 8; .
+	pub const ProxyDepositBase: Balance = deposit(1, 8);
+	// Additional storage item size of 33 bytes.
+	pub const ProxyDepositFactor: Balance = deposit(0, 33);
+	pub const MaxProxies: u16 = 32;
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug)]
+pub enum ProxyType {
+	Any,
+	NonTransfer,
+	Governance,
+	Staking,
+}
+impl Default for ProxyType { fn default() -> Self { Self::Any } }
+impl InstanceFilter<Call> for ProxyType {
+	fn filter(&self, c: &Call) -> bool {
+		match self {
+			ProxyType::Any => true,
+			ProxyType::NonTransfer =>
+				matches!(c, Call::Balances(..) | Call::Vesting(pallet_vesting::Call::vested_transfer(..))),
+			ProxyType::Governance => matches!(c, Call::Democracy(..) | Call::Council(..) | Call::Society(..)
```

Comment:
When thinking about ways I could transfer funds without using balances or vesting, I thought about calls that allow us to repatriate reserved.

Reap-preimage in democracy is a way a user can reserve funds on behalf of one account, and then move those funds into another account. Report defunct voter in elections too. Same with Indices "transfer".

Reply:
see above